ui/STTListMenu.qml

```diff
@@ -151,7 +151,7 @@ Item {
 
                     onClicked: {
                         Mycroft.SoundEffects.playClickedSound(Qt.resolvedUrl("sounds/clicked.wav"))
-                        triggerGuiEvent("mycroft.device.confirm.stt", {"engine": model.engine, "display_name": model.display_name, "offline": model.offline})
+                        triggerGuiEvent("mycroft.device.confirm.stt", model)
```

Comment:
This is a breaking change; handler method in skill receives empty dict data

Reply:
i thought the full model would auto expand, seems we need to manually add all keys here then to ensure the final hash matches, this was not intended to be breaking, rather to add extra keys and ensure none are missing for hash step

Reply:
this needs to exactly match
```python
{"plugin_name": plugin_display_name,
"display_name": config.get("display_name", " "),
"offline": config.get("offline", False),
"lang": lang,
"engine": engine}
```